src/main/kotlin/no/nav/syfo/dinesykmeldte/api/DineSykmeldteApi.kt

```diff
@@ -0,0 +1,15 @@
+package no.nav.syfo.dinesykmeldte.api
+
+import io.ktor.application.call
+import io.ktor.http.HttpHeaders
+import io.ktor.response.respond
+import io.ktor.routing.Route
+import io.ktor.routing.get
+import no.nav.syfo.dinesykmeldte.service.DineSykmeldteService
+
+fun Route.registerDineSykmeldteApi(dineSykmeldteService: DineSykmeldteService) {
+    get("dinesykmeldte") {
```

Comment:
bør kanskje starte med /?

Reply:
Bør apiet merkes med at det er tilsvarende format som syforest ga..? Det ser iallefall ut til å være det som returneres :) Det kan gjøre det enklere å rydde når vi en dag blir kvitt gammel frontend :)

Reply:
trenger nok ikke "/" det blir ikke helt det samme formatet som syforest gir egentlig, har har lagt det under "DineSykmeldte" så vi kan kalle det for noe annet når vi lager ny frontend :D